integration_tests/e2e/abilityToWork.cy.ts

```diff
@@ -114,10 +120,12 @@ context('Ability to work page', () => {
 
     abilityToWorkPage.submitButton().click()
 
-    cy.url().should('include', 'check-your-answers')
+    const checkYourAnswersPage = Page.verifyOnPage(CheckYourAnswersPage)
```

Comment:
The previous test only asserted the user had landed on the Check Your Answers page (ie. a GET). Thats all it did

I've changed it so that it submits that page, and we expect to be on a PLP page (via the stub `stubRedirectToPlpAfterCreateInduction` and `stubPlpEducationAndTrainingPageUi`)

Also worth noting that this test (and most others) are Create journey tests, not Update.

I've updated and added to the Update tests below 👇